shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FaultResult.java

```diff
@@ -37,8 +50,25 @@ public String getId() {
     /**
      * Value of the 3-phase short-circuit current for this fault (in A).
      */
-    public float getThreePhaseFaultCurrent() {
+    public double getThreePhaseFaultCurrent() {
         return threePhaseFaultCurrent;
     }
 
+    /**
+     * List of contributions to the three phase fault current of each connectable connected to the equipment
+     */
+    public List<FeederResult> getFeederResults() {
+        return feederResults;
+    }
+
+    public double getFeederCurrent(String feederId) {
+        double current = 0;
```

Comment:
Here, if the `id` does not exist, the current is 0. Is it what we want? Wouldn't we rather have `Double.NaN`?